admin/app/components/solidus_admin/sidebar/component.html.erb

```diff
@@ -31,7 +31,7 @@
     </div>
     <%= render component("sidebar/account_nav").new(
       user_label: helpers.current_solidus_admin_user.email,
-      account_path: solidus_admin.account_path,
+      account_path: account_path,
```

Comment:
It would be easier to understand where this method comes from by prefixing it with `routes`, like with `helpers`.

```suggestion
      account_path: routes.account_path,
```